regolith/helpers/u_finishprumhelper.py

```diff
@@ -18,6 +18,10 @@ def subparser(subpi):
     subpi.add_argument("-d", "--database",
                        help="The database that will be updated.  Defaults to "
                             "first database in the regolithrc.json file.")
+    subpi.add_argument("--pub_sum", type = str,
+                       help="The public summary that is mandatory for finishing a prum.")
+    subpi.add_argument("--pro_sum", type = str,
+                       help="The professional summary that is mandatory for finishing a prum.")
```

Comment:
please change to:
"The professional summary. This may be used in, for example, a progress report to a funding agency, or in your cv, etc.. It should capture in a single paragraph the  main context, result and importance in more technical language with the target audience of an expert in an adjacent field.  This is required to finish a prum